06-building-a-program-start-to-finish/setting-up-linters/Output/eslint.md

```diff
@@ -0,0 +1 @@
+https://github.com/andela-fomokaro/Inverted-Index-Project/blob/features/Pictures%20of%20eslint%20error%20violation%20and%20houndci/Screen%20Shot%202017-03-26%20at%207.43.05%20AM.png
```

Comment:
What output is this?
This ReadMe file is not properly formatted. 
Multiple links for a single output should be in one ReadMe file.